### PR TITLE
Handle encryption when adding host to zabbix server

### PR DESCRIPTION
**Pull request by sblaisot**
**Description of PR**

This PR adds the Encryption properties when creating the new host in zabbix server, according to agent's configuration. This avoid creating the host without encryption when the agent is configured to only accept encrypted connections.

**Type of change**

Bugfix Pull Request

### CODE REVIEW

**Comment by sblaisot** [comment]
Duplicate with #148

File: tasks/main.yml [hunk 1]
```diff
@@ -223,6 +223,12 @@
     inventory_mode: "{{ zabbix_inventory_mode }}"
     interfaces: "{{ zabbix_agent_interfaces }}"
     visible_name: "{{ zabbix_visible_hostname|default(zabbix_agent_hostname) }}"
+    tls_psk: "{{ zabbix_agent_tlspsk_secret|default(omit) }}"
+    tls_psk_identity: "{{ zabbix_agent_tlspskidentity|default(omit) }}"
+    tls_issuer: "{{ zabbix_agent_tlsservercertissuer|default(omit) }}"
+    tls_subject: "{{ zabbix_agent_tlsservercertsubject|default(omit) }}"
+    tls_connect: "{{ zabbix_server_tlsconnect_value[zabbix_agent_tlsaccept if zabbix_agent_tlsaccept else 'unencrypted'] }}"
+    tls_accept: "{{ zabbix_server_tlsaccept_value[zabbix_agent_tlsconnect if zabbix_agent_tlsconnect else 'unencrypted'] }}"
```
Comment (dsteinkopf) [inline tasks/main.yml]: Values tls_connect and tls_accept have to be swapped. Reason: tls_accept is from the zabbix server's point of view and zabbix_agent_tlsaccept is from the agent's point of view.

Comment (sblaisot) [reply tasks/main.yml]: they ARE swapped ;)

**Comment by sblaisot** [comment]
ok, PR reoppened.

I included README example from @dsteinkopf 's PR #155 and removed unneeded variable like in @skazochnik97  's PR #148 

Ready to merge.

File: tasks/main.yml (hunk 1 above)
Comment (dsteinkopf) [reply tasks/main.yml]: Ah yes sorry, you're right.

**Comment by dj-wasabi** [comment]
@sblaisot Thanks!

**Pull request merged**